.dockerignore

```diff
@@ -1,6 +1,5 @@
 .dockerignore
 Dockerfile
-.git/
```

Comment:
Hmm, why is this change? I seem to have missed this in the PR #1090 review.

Reply:
Because now when we compile the binary, we need access to git version and I access it via `git` command (there is a cargo to do that)